.github/workflows/valid-links.yml

```diff
@@ -14,6 +14,7 @@ jobs:
       - run: |
           git fetch --prune origin master
           git checkout $GITHUB_SHA
+          git rebase origin/master
```

Comment:
This step is required to have `git diff` only output the actual changes made in this branch. Indidentally, this was probably the reason why the link checker failed in #234, as the diff could show files that were deleted.